src/github/issueModel.ts

```diff
@@ -0,0 +1,129 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Remote } from '../common/remote';
+import { GitHubRepository } from './githubRepository';
+import { IAccount, Issue, GithubItemStateEnum, IMilestone } from './interface';
+
+export class IssueModel {
+	public id: number;
+	public graphNodeId: string;
+	public githubNumber: number;
```

Comment:
nit: I think just `number` would be fine

Reply:
Changed!